core/src/main/java/org/stellar/anchor/auth/JwtService.java

```diff
@@ -211,7 +206,21 @@ public <T extends AbstractJwt> T decode(String cipher, Class<T> cls)
     }
   }
 
-  private String toBase64OrNull(String value) {
-    return value == null ? null : Base64.encodeBase64String(value.getBytes(StandardCharsets.UTF_8));
+  @SneakyThrows
+  public Jws<Claims> getHeaderJwt(String signingKey, String cipher) {
+    var factory = KeyFactory.getInstance("Ed25519");
```

Comment:
How does this relate to SEP10
[```{
  "alg": "EdDSA"
}```](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0010.md#request-header)

Is this verified by the JWT library?

Reply:
Yes